src/directives/ClickToAccept.tsx

```diff
@@ -78,24 +94,26 @@ const ClickToAccept = ({ directive, status, mutate }: DirectiveProps) => {
                 }
 
                 const data = clickToAcceptResponse.data[0];
-                jobStatusPoller(
-                    jobStatusQuery(data),
-                    async () => {
-                        trackEvent(`${directiveName}:Complete`, directive);
-                        void mutate();
-                    },
-                    async (payload: any) => {
-                        trackEvent(`${directiveName}:Error`, directive);
-                        setSaving(false);
-                        setServerError(payload.job_status.error);
-                    }
-                );
+                pollForDirective(data);
             }
         },
     };
 
     useMount(() => {
         trackEvent(`${directiveName}:Viewed`);
+
+        // The user may have reloaded while they are waiting on the directive
+        //  to go through. If so - we can just keep them on the loading view
+        if (directive && waiting) {
+            trackEvent(`${directiveName}:AutoPolling`);
+            setShowErrors(false);
+            setSaving(true);
+            pollForDirective({
+                logs_token: directive.logs_token,
+                directive_id: directive.directive_id,
+                id: directive.id,
+            });
+        }
```

Comment:
If during mount we have all this we should be good to kick off polling and waiting again.